mon-pix/app/components/challenge/challenge-statement/focused-tooltip.js

```diff
@@ -0,0 +1,54 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class FocusedTooltip extends Component {
+  @service currentUser;
+  @tracked shouldDisplayTooltip = false;
+
+  constructor() {
+    super(...arguments);
+    this._showTooltip();
+  }
+
+  get isFocusedChallenge() {
+    return this.args.challenge.focused;
+  }
+
+  _showTooltip() {
+    if (this.isFocusedChallenge && this.currentUser.user && !this.currentUser.user.hasSeenFocusedChallengeTooltip) {
+      this.shouldDisplayTooltip = true;
+    }
+    else if (this.isFocusedChallenge && this.currentUser.user && this.currentUser.user.hasSeenFocusedChallengeTooltip) {
+      this._notifyChallengeTooltipIsClosed();
+    }
+    else if (this.isFocusedChallenge) {
+      this._notifyChallengeTooltipIsClosed();
+    }
+  }
+
+  @action
+  confirmInformationIsRead() {
+    this.shouldDisplayTooltip = false;
+    this._notifyChallengeTooltipIsClosed();
+  }
+
+  _notifyChallengeTooltipIsClosed() {
+    this.args.onTooltipClose();
+  }
+
+  @action
+  toggleTooltip(value) {
+    if (this.isFocusedChallenge && this.currentUser.user && this.currentUser.user.hasSeenFocusedChallengeTooltip) {
+      this.shouldDisplayTooltip = value;
+    }
+    if (this.isFocusedChallenge && !this.currentUser.user) {
+      this.shouldDisplayTooltip = value;
+    }
```

Comment:
Est-ce que

```suggestion
      this.shouldDisplayTooltip = value;
```

ne suffit pas dans ce cas ?

Reply:
Vu en review : besoin de séparer les cas